components/List/README.md

```diff
@@ -50,9 +50,8 @@ Below is an example:
   - [Importing](#importing)
 - [Usage](#usage)
   - [Typical use](#typical-use)
-- [Extensions](#extensions)
-  - [Color Theming](#color-theming)
-  - [Typography Theming](#typography-theming)
+- [Theming](#theming)
+  - [List Item Theming](#list-item-theming)
```

Comment:
I'm not sure we need a sub-item here. I'd recommend deleting the "List Item Theming" heading.